interchain-token-service/src/primitives.rs

```diff
@@ -56,13 +64,13 @@ pub enum ItsHubMessage {
     /// ITS edge source contract -> ITS Hub
     SendToHub {
         /// True destination chain of the ITS message
-        destination_chain: ChainName,
+        destination_chain: ChainNameRaw,
```

Comment:
Are you planning to change this? I thought core was able to accept a normalized name, and do the conversion if necessary. I was under the impression that ChainNameRaw was only needed for the source chain for messages to amplifier

Reply:
Yes, I changed it back to `ChainName` to mimic GMP